EDITOR_SETUP.md

```diff
@@ -158,3 +158,36 @@ according to our configuration.
 [ra_vsc]: https://marketplace.visualstudio.com/items?itemName=matklad.rust-analyzer
 [rustfmt]: https://github.com/rust-lang/rustfmt
 [vsc]: https://code.visualstudio.com/
+
+## TypeScript Setup
+
+In an effort to maintain consistency among the files in our front-end codebase, we use the VS Code extension, "Prettier". The reason why we use a consistent formatter on our front-end is because certain elements require no whitespace between tags. Sometimes auto-formatters add extra spaces, which may occasionally result in bugs. 
```

Comment:
I'd prefer to not have huge lines, rather split before 80 characters or something.

Reply:
Done!